Makefile

```diff
@@ -1,18 +1,22 @@
 PROTO_DIR = Tests/Protobuf
 PROTO_GEN_DIR = Tests/CombineGRPCTests/Generated
+PROTOC_PATH= protoc/bin/protoc
```

Comment:
Sorry, I don't think it's a good idea to add binaries of unknown version/provenance into the repo. Can you remove protoc and plugins?

Reply:
Sure. I'll delete it right away. 

But in my opinion, I think it is better to use binaries included from the repo to compile the `.proto` files to reduce build error if the local `protoc` compiler version is not correct.